source/codegen/metadata/nidmm/enums.py

```diff
@@ -68,48 +44,30 @@
     'ApertureTimeUnits': {
         'values': [
             {
-                'documentation': {
-                    'description': 'Seconds'
-                },
                 'name': 'NIDMM_VAL_SECONDS',
                 'value': 0
             },
             {
-                'documentation': {
-                    'description': 'Powerline Cycles'
-                },
-                'name': 'NIDMM_VAL_POWER_LINE_CYCLES',
+               'name': 'NIDMM_VAL_POWER_LINE_CYCLES',
                 'value': 1
             }
         ]
     },
     'AutoZero': {
         'values': [
             {
-                'documentation': {
-                    'description': 'The drivers chooses the AutoZero setting based on the configured function  and resolution.'
-                },
                 'name': 'NIDMM_VAL_AUTO_ZERO_AUTO',
                 'value': -1
             },
             {
-                'documentation': {
-                    'description': 'Disables AutoZero.'
-                },
                 'name': 'NIDMM_VAL_AUTO_ZERO_OFF',
                 'value': 0
             },
             {
-                'documentation': {
-                    'description': 'The DMM internally disconnects the input signal following each measurement  and takes a zero reading. It then subtracts the zero reading from the  preceding reading.'
-                },
-                'name': 'NIDMM_VAL_AUTO_ZERO_ON',
+               'name': 'NIDMM_VAL_AUTO_ZERO_ON',
```

Comment:
The spacing changed for 'name'. Make it line up with 'value' beneath.

Reply:
fixed